src/util/tracking.ts

```diff
@@ -106,13 +108,13 @@ export function logEvent(event: TrackingEventName, values: TrackingValues = {})
  * Send a raw event to Firebase.
  */
 async function logToFirebase(name: TrackingEventName, values: TrackingValues) {
-  const { accountDate, currencyCode, dollarValue, installerId, pluginId, error } = values
+  const { accountDate, currencyCode, dollarValue, installerId, pluginId, error, ...rest } = values
 
   // @ts-expect-error
   if (!global.firebase) return
 
-  // Persistent params:
-  const params: any = { edgeVersion: getVersion(), isFirstOpen: await getIsFirstOpen() }
+  // Persistent & Unchanged params:
```

Comment:
All the logic that modifies the params should be moved to `logEvent` so it hits both firebase and the util server.

Add this as another commit that cleans up the logic